public/app/core/components/code_editor/code_editor.ts

```diff
@@ -66,6 +52,34 @@ function link(scope: any, elem: any, attrs: any) {
 
   // Initialize editor
   const aceElem = elem.get(0);
+  const { default: ace } = await import(/* webpackChunkName: "brace" */ 'brace');
+  // @ts-ignore
```

Comment:
What are we ignoring here?  Lack of assignment?

Reply:
meant to reply to this earlier - typescript complains about these imports not having types, basically. maybe there's a better solution than all these ignore comments?

Reply:
Yeah, I see that now. I don't think tsc supports ignore blocks. What you could do instead is to create a declaration file for brace with content like this:

```
declare module 'brace/*' {}
```

This should help.

Reply:
nice, i'll try this